tests/migration/reset_scc_addons_values.pm

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Swap the values of SCC_ADDONS before migration.
+#
+# In migration maintenance updates tests, SCC_ADDONS is used for both
+# registering original system and adding target system modules maintenance
+# updates repos. But for SLE 12 and SLE 15, the modules name are different.
+# So swap SCC_ADDONS values to upgraded SLE 15 modules before migration.
+# And re-create incidents repos based on the swapped SCC_ADDONS values.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    set_var('SCC_ADDONS', get_var('SCC_ADDONS_2'));
```

Comment:
Once you move the other part to a new file, you can rename this file to `prepare_scc_addons.pm`.
I'm not really sure at the moment how (if we can?) combine some logic here without conditional logic to act regardless we swap with the target or with the origin, but for now, would should suffice.

Reply:
done, thanks.